src/main/scala/com/lightbend/conductr/sbt/ConductRPlugin.scala

```diff
@@ -45,26 +47,165 @@ object ConductrPlugin extends AutoPlugin {
+      case InstallationData(bundleName, bundle, bundleConfigFile) =>
+        val bundleIdEnv = bundleName.replaceAll("\\W", "_").toUpperCase + "_BUNDLE_ID"
+        val bundleArg = InstallationData.nameOrPath(
+          bundle match {
+            case Left(b)  => Left(b)
+            case Right(b) => Right(installPath.relativize(b))
+          }
+        )
+        val bundleConfigArg = bundleConfigFile.fold("")(installPath.relativize(_) + " ")
+        s"""
+           |echo "Deploying $bundleName..."
+           |$bundleIdEnv=$$(conduct load $bundleArg $bundleConfigArg--long-ids -q)
+           |conduct run $${$bundleIdEnv} --no-wait -q
+                 """.stripMargin
+    }
+    IO.write(
+      installationScript,
+      installationScriptContents.mkString(
+        s"""#!/usr/bin/env bash
+            |cd "$$( dirname "$${BASH_SOURCE[0]}" )"
+            |""".stripMargin,
+        "",
+        """
+          |echo 'Your system is deployed. Running "conduct info" to observe the cluster.'
+          |echo 'Type just "conduct" for help.'
+          |conduct info
+          |""".stripMargin
+      ),
+      Utf8
+    )
+    println("\nYour system's ConductR installation script: " + installationScript)
+    installationScript.setExecutable(true)
+    installationScript
+  }
+
+  private def installTask(): Def.Initialize[Task[Unit]] = Def.task {
+    withProcessHandling {
+      val nrOfContainers = "docker ps -q --filter='name=cond-'".lines_!.size
+      if (nrOfContainers > 0) {
+        println("Restarting ConductR to ensure a clean state...")
+        "docker exec cond-0 rm /opt/conductr/conf/seed-nodes".! // Allow the first node to form the new cluster
+        for (n <- 0 until nrOfContainers) s"docker restart -t 0 cond-$n".!
+      } else {
+        throw new IllegalStateException("Please first start the sandbox using 'sandbox run'.")
+      }
+    }(sys.error("There was a problem re/starting the sandbox."))
+
+    withProcessHandling {
+      waitForConductr(20.seconds)
+
+      installationData.value.foreach {
+        case InstallationData(bundleName, bundle, bundleConfigFile) =>
+          println(s"Deploying $bundleName...")
+          val bundleArg = InstallationData.nameOrPath(bundle)
+          val bundleConfigArg = bundleConfigFile.fold("")(_ + " ")
+          s"conduct load $bundleArg $bundleConfigArg--long-ids -q".lines_!.headOption match {
+            case Some(bundleId) =>
+              if (s"conduct run $bundleId --no-wait -q".! != 0)
+                throw new IllegalStateException("The bundle could not run")
+            case None =>
+              throw new IllegalStateException("The bundle could not be loaded")
+          }
+      }
+
+    }(sys.error("There was a problem installing the project to ConductR."))
+
+    println
+    println("""Your system is deployed. Running "conduct info" to observe the cluster.""")
+    println("""Type just "conduct" for help.""")
```

Comment:
Not sure if we need the line "echo 'Type just "conduct" for help.'". At that point in time the user should familiar with the conduct command. Also he might want to use `conduct logs`, depending on the situation.

If we stick with the line I would rename it to: `echo 'Type "conduct" for help.'`